sounds/views.py

```diff
@@ -244,6 +245,16 @@ def sound(request, username, sound_id):
     is_explicit = sound.is_explicit and (not request.user.is_authenticated() \
                         or not request.user.profile.is_adult)
 
+    # Show modal asking users for donations after clicking in download
```

Comment:
Now the content of the modal is set on template so after_download_modal is not used anymore